Source/Scene/Cesium3DTileset.js

```diff
@@ -349,6 +329,36 @@ define([
         when(tile.readyPromise).then(removeFunction).otherwise(removeFunction);
     }
 
+    function selectTileWithTilesetContent(tiles3D, selectedTiles, tile, fullyVisible, frameState, replace) {
+        // 1) If its children are not loaded, load the subtree it points to and then select its root child
+        // 2) If its children are already loaded, select its (root) child since the geometric error of it is
+        //    same as this tile's
+        var childrenLength = tile.children.length;
+        var contentUrl = tile._header.content.url;
+
+        // If the subtree has already been added and child
+        // content requested, select the child and continue
+        if ((childrenLength > 0) && (tile.numberOfChildrenWithoutContent === 0)) {
+            selectTile(selectedTiles, tile.children[0], fullyVisible, frameState);
+            return;
+        } else if (replace) {
+            // Otherwise, select the parent tile, to avoid showing an empty space
+            // Note that tile.parent must always be defined here --
+            // tile should never be the root of the tileset AND
+            // contain another tileset
+            selectTile(selectedTiles, tile.parent, fullyVisible, frameState);
+        }
+
+        // Request the tile's tileset.
+        if (childrenLength === 0 && !tile.tilesetLoading) {
```

Comment:
1. You're right, it would be better without this limitation and it shouldn't be hard implementation-wise. Will have a look at this (probably not until next week).

2. That would definitely make it clearer. I did consider this for a moment but got paralyzed by choice :) Will have this changed.